test/e2e/common_test.go

```diff
@@ -2,35 +2,57 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package e2e
+package e2e_test
 
 import (
 	"bytes"
-	"context"
 	"encoding/json"
 	"fmt"
 	"io"
 	"os"
 	"strings"
-	"time"
 
 	. "github.com/onsi/gomega"
-	framework "github.com/operator-framework/operator-sdk/pkg/test"
-	"knative.dev/pkg/apis"
 
+	"golang.org/x/net/context"
```

Comment:
Should we keep using the standard `context` package?

I searched in Google, most of the issue is swtich `golang.org/x/net/context` to standard `context, like: 
https://github.com/grpc-ecosystem/go-grpc-middleware/issues/175

Reply:
That sounds like a mistake when creating the import statements. I will fix that. Thanks for finding that.